lcdmain.py

```diff
@@ -2,21 +2,21 @@
     XBMC LCDproc addon
     Copyright (C) 2012 Team XBMC
     Copyright (C) 2012 Daniel 'herrnst' Scheller
-    
+
```

Comment:
Please rebase on the leia branch (which will eventually become master soon). I have a cosmetic fixup patch in there that already gets rid of all the whitespace whoes. Won't comment on all the other occurences.